### PR TITLE
checker: make optional arguments in func call an error

### PR DESCRIPTION
**Pull request by zakuro9715**

### CODE REVIEW

File: vlib/v/checker/tests/optional_fn_err.vv [hunk 1]
```diff
@@ -1,15 +1,50 @@
+import json
```
Comment (spytheman) [inline vlib/v/checker/tests/optional_fn_err.vv:1]: That is not needed/used? Can it be removed?

File: vlib/v/checker/tests/optional_fn_err.vv [hunk 2]
```diff
@@ -1,15 +1,50 @@
...
+	v.value += n
+}
+
+fn twice(n int) int {
+	return n * 2
+}
+
 fn main() {
-	// Calling foo() without ? or an or block, should be an error.
+	// Callingd foo() without ? or an or block, should be an error.
```
Comment (EPonyA) [inline vlib/v/checker/tests/optional_fn_err.vv:26]: Was this change intentional?
